cmd/kube-controller-manager/app/options/options.go

```diff
@@ -165,6 +167,7 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.TerminatedPodGCThreshold, "terminated-pod-gc-threshold", s.TerminatedPodGCThreshold, "Number of terminated pods that can exist before the terminated pod garbage collector starts deleting terminated pods. If <= 0, the terminated pod garbage collector is disabled.")
 	fs.DurationVar(&s.HorizontalPodAutoscalerSyncPeriod, "horizontal-pod-autoscaler-sync-period", s.HorizontalPodAutoscalerSyncPeriod, "The period for syncing the number of pods in horizontal pod autoscaler.")
 	fs.DurationVar(&s.DeploymentControllerSyncPeriod, "deployment-controller-sync-period", s.DeploymentControllerSyncPeriod, "Period for syncing the deployments.")
+	fs.DurationVar(&s.HealthCheckControllerSyncPeriod, "health-check-controller-sync-peroid", s.HealthCheckControllerSyncPeriod, "Period for checking the the health of all API servers (set to 0 to disable)")
```

Comment:
The name of this needs to actually correspond to what it is doing - "apiserver-health-check-period"